init.lua

```diff
@@ -68,6 +78,30 @@ local function get_destination(player, stack)
 	end
 end
 
+local function teleport_above(playername, target, counter)
```

Comment:
this entire block is indented 1 tab too far

Reply:
fixed